README.md

```diff
@@ -59,11 +59,17 @@ Use the Test Runner Tool in your Squeak Image and select all *AlgernonTests*. No
 
 *  [Ann Katrin Kuessner](https://github.com/annkatrinkuessner)
 *  [Ben Bals](https://github.com/BenBals)
+*  [Benedikt Weber](https://github.com/bewee)
+*  [Franz Sauerwald](https://github.com/FranzSw)
 *  [Jannis Rosenbaum](https://github.com/sinnaj-r)
 *  [Jonathan Schneider](https://github.com/jonaschn)
 *  [Kai Robert Kirsten](https://github.com/robertkirsten)
 *  [Kay Erik Jenß](https://github.com/kej-jay)
+*  [Laura Meister](https://github.com/dieknolle3333)
 *  [Louis Kirsch](https://github.com/timediv)
+*  [Niklas Mohrin](https://github.com/niklasmohrin)
+*  [Nils Lißner](https://github.com/TheGrayStone)
+*  [Olli Wehr](https://github.com/Olliwehr)
```

Comment:
Uff 😄

Reply:
All intended

Reply:
Do you want to troll me even further? xD